src/components/Block.jsx

```diff
@@ -20,20 +21,31 @@ module.exports = React.createClass({
     return app.refine('blockTypes').find(i => i.id === block.type)
   },
 
+  setMenuItems(component) {
+    if ('getMenuItems' in component) {
```

Comment:
Do you want to make an additional check that this is a function, or is this sufficient?